pkg/cluster/nodes/node.go

```diff
@@ -279,6 +286,28 @@ func (n *Node) IP() (ip string, err error) {
 	return ip, nil
 }
 
+// IPPrefix returns the Subnet prefix of the node
+func (n *Node) IPPrefix() (ipPrefix string, err error) {
+	// use the cached version first
+	cachedIPPrefix := n.cache.IPPrefix()
+	if cachedIPPrefix != "" {
+		return cachedIPPrefix, nil
+	}
+	// retrive the subnet prefix of the node using docker inspect
+	lines, err := docker.Inspect(n.name, "{{range .NetworkSettings.Networks}}{{.IPPrefixLen}}{{end}}")
+	if err != nil {
+		return "", errors.Wrap(err, "failed to get file")
```

Comment:
Wrong error message?

Reply:
yeah, I rushed it a bit, copy paste thing